packages/i18n/src/messages/localizedConstants.tsx

```diff
@@ -558,6 +558,18 @@ export const poolCandidateStatuses = defineMessages({
     id: "R2BWry",
     description: "The pool candidate's status is Screened Out Application",
   },
+  [PoolCandidateStatus.ScreenedOutNotInterested]: {
```

Comment:
@Jerryescandon These look a bit awkward since the other _screened out_ statuses are not hyphenated.  Are you sure want them named like this?
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/10392c14-7b30-4c6a-9b3d-509b51d0d722)

Reply:
Thanks @Jerryescandon , I asked in standup and it was decided to add dashes to the other statuses. These were added in [add dashes to existing screened out statuses](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7296/commits/c2e6098b69f244d86698c3bd7d712408dc539024)